### PR TITLE
add nokov support

### PR DESCRIPTION
**Pull request by duguguang**
add nokov support

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@ option(LIBMOTIONCAPTURE_ENABLE_QUALISYS "Enable Qualisys" ON)
 option(LIBMOTIONCAPTURE_ENABLE_OPTITRACK "Enable Optitrack" ON)
 option(LIBMOTIONCAPTURE_ENABLE_OPTITRACK_CLOSED_SOURCE "Enable Optitrack (Closed Source)" OFF)
 option(LIBMOTIONCAPTURE_ENABLE_VICON "Enable Vicon" ON)
+option(LIBMOTIONCAPTURE_ENABLE_NOKOV "Enable Nokov" ON)
```
Comment (whoenig) [inline CMakeLists.txt:10]: This should be disabled by default, since the user needs to manually find and copy the SDK

**Review by whoenig — CHANGES_REQUESTED** [review]
Very nice work - thanks! Since the SDK currently doesn't seem to be publicly available, I suggest disabling this by default. Note that this also means that this code will not be built in continuous integration (and thus not be part of any official pip package for the Python version).

File: CMakeLists.txt (hunk 1 above)
Comment (duguguang) [reply CMakeLists.txt]: Thank you for reminding.

**Comment by whoenig** [comment]
Thanks!

**Pull request merged**